assigner/tqdm.py

```diff
@@ -0,0 +1,47 @@
+import contextlib
+import sys
+from tqdm import tqdm as _tqdm
+
+class DummyTqdmFile(object):
+    """Dummy file-like that will write to tqdm"""
+    file = None
+    def __init__(self, file):
+        self.file = file
```

Comment:
I recognize that there are three files here but they're all named the same thing. Can we rename them?